Modify API to get attractions

ann-yachen opened the pull request:
Ann (陳璵安)

Modify API to get attractions
- Result is ok. Modify /api/attractions to get all attractions from database by SQL command LIMIT

Deploy python flask server on AWS EC2 instance
- http://35.155.30.220:3000/api/attractions?page=1
- http://35.155.30.220:3000/api/attractions?page=0&keyword=藝文館所
- http://35.155.30.220:3000/api/attractions?page=0&keyword=北
- http://35.155.30.220:3000/api/attraction/10

ann-yachen committed:
Add APIs for Attractions and Categories

- Add /api/attraction to get attractions from database which can be filtered by keyword
- Add /api/attraction/attractionId to get data of attraction by id
- Add /api/categories to get categories of attractions

ann-yachen committed:
Modify API to get attractions
- Modify /api/attractions to get all attractions from database by SQL command LIMIT

cwpeng reviewed (changes requested):
Try to use `LIMIT` in keyword search query.

You can re-request review from the right side of this page.

ann-yachen committed:
Modify API to get attractions by keyword
- Modify /api/attractions to get attractions from database by keyword by SQLcommand

cwpeng reviewed (approved):
OK. Well Done. :)

ann-yachen commented:
Ann (陳璵安)

Modify API to get attractions

- Modify /api/attractions to get attractions from database by keyword with SQL command LIMIT

Deploy python flask server on AWS EC2 instance

- http://35.155.30.220:3000/api/attractions?page=1
- http://35.155.30.220:3000/api/attractions?page=0&keyword=藝文館所
- http://35.155.30.220:3000/api/attractions?page=0&keyword=北
- http://35.155.30.220:3000/api/attraction/10
